osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/infra/jms/protocol/outgoing/ProtocolRequestMessageSender.java

```diff
@@ -5,7 +5,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package org.opensmartgridplatform.core.infra.jms.protocol;
+package org.opensmartgridplatform.core.infra.jms.protocol.outgoing;
```

Comment:
Maybe the package name should be org.opensmartgridplatform.core.infra.jms.protocol.outbound